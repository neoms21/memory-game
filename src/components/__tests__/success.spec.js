import React from 'react';
import Renderer from 'react-test-renderer';
import 'jest-styled-components';
import Success from '../success';

describe('Success tests', () => {
  it('should render null', () => {
    const wrapper = Renderer.create(<Success />);

    expect(wrapper).toMatchSnapshot();
  });

  it('should render success', () => {
    const wrapper = Renderer.create(<Success isGameWon={true} />);

    expect(wrapper).toMatchSnapshot();
  });
});
