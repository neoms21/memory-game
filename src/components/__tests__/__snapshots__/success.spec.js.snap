// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success tests should render null 1`] = `null`;

exports[`Success tests should render success 1`] = `
.c0 {
  background: #21b241;
  color: #fff;
  padding: 8px;
  margin-bottom: 10px;
}

<div
  className="c0"
  id="success"
>
  Congratulations!!! you won the game.
</div>
`;
